src/hub.js

```diff
@@ -624,6 +626,11 @@ document.addEventListener("DOMContentLoaded", async () => {
   const scene = document.querySelector("a-scene");
   scene.setAttribute("shadow", { enabled: window.APP.quality !== "low" }); // Disable shadows on low quality
 
+  // HACK - Trigger initial batch preparation with an invisible object
```

Comment:
Why is this necessary / desirable?

Reply:
This is only tangentially related to the emoji feature. Currently, the batching system compiles its shader code the very first time you load an image into a room. This causes a large (~3s) hitch on both Firefox and Chrome. Obviously this affects the emoji picture, since we're spawning the emoji in the world as a regular old `media` object, which uses the batching system. So if an emoji is the first image in the room, it would cause that hitch.

This hack just adds an invisible image asset to the scene so that the batching shaders get initialized during room load.

@netpro2k Perhaps you have strong opinions about this hack.